src/Arcus.Scripting.ApiManagement/Scripts/Create-AzApiManagementUserAccountsFromConfig.ps1

```diff
@@ -0,0 +1,215 @@
+      "type": "array",
+      "title": "The configuration JSON schema",
+      "$defs": {},
+      "prefixItems": [
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "firstName": {
+              "type": "string"
+            },
+            "lastName": {
+              "type": "string"
+            },
+            "userId": {
+              "type": "string"
+            },
+            "mailAddress": {
+              "type": "string"
+            },
+            "sendNotification": {
+              "type": "boolean"
+            },
+            "confirmationType": {
+              "type": "string",
+              "enum": ["signup", "invite"]
+            },
+            "note": {
+              "type": "string"
+            },
+            "groups": {
+              "type": "array",
+              "prefixItems": [
+                {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "properties": {
+                    "id": {
+                      "type": "string"
+                    },
+                    "displayName": {
+                      "type": "string"
+                    },
+                    "description": {
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "id",
+                    "displayName"
+                  ]
+                }
+              ]
+            },
+            "subscriptions": {
+              "type": "array",
+              "prefixItems": [
+                {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "properties": {
+                    "id": {
+                      "type": "string"
+                    },
+                    "displayName": {
+                      "type": "string"
+                    },
+                    "scope": {
+                      "type": "string"
+                    },
+                    "allowTracing": {
+                      "type": "boolean"
+                    }
+                  },
+                  "required": [
+                    "id",
+                    "displayName",
+                    "scope",
+                    "allowTracing"
+                  ]
+                }
+              ]
+            }
+          },
+          "required": [
+            "firstName",
+            "lastName",
+            "mailAddress",
+            "sendNotification",
+            "confirmationType"
+          ]
+        }
+      ]
+    }
+'@
+
+if (-not (Get-Content -Path $ConfigurationFile -Raw | Test-Json -Schema $schema -ErrorAction SilentlyContinue)) {
+    throw "Cannot apply user configuration based on JSON configuration file because the file does not contain a valid JSON configuration file."
```

Comment:
Those are some great exception messages!
Maybe we can also include the action we were trying to take? Like: `Cannot apply user configuration to Azure API Management service '$ServiceName' based on JSON configuration file...`

Just that if this is used in a script, we immediately know where the exception is caused, instead of looking at the line number.